apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -315,3 +315,11 @@ function load_wpcom_block_editor_sidebar() {
 	}
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_wpcom_block_editor_sidebar' );
+
+/**
+ * Coming soon
+ */
+function load_coming_soon() {
```

Comment:
Will we need any feature flag here to prevent the module loading? Or are we just relying on the non-existence of the coming soon options.